examples/pipeline-demo.yaml

```diff
@@ -9,18 +9,14 @@ spec:
       brigade.js: |-
         const { events, Job } = require("brigadier");
 
-        events.on("github.com/brigadecore/brigade/cli:exec", (e) => {
-          let fooJob = new Job("foo", "debian:latest");
+        events.on("github.com/brigadecore/brigade/cli", "exec", async event => {
```

Comment:
Non/blocking; just an idea: Should we add a line where we demonstrate calling a job's `logs()` method?

(Ignore if we already do this in another example, etc.)

Reply:
The prototype had some such examples. I haven't ported over all examples yet.

It does work, but I should note that I'm questioning whether that's a good/practical thing for people to do.

Jobs produce logs. Those logs are already accessible using the API or CLI. I can't figure out why anyone would need to access them from inside the worker. If it's just to dump them, that's kind of pointless, because, as I said, the job logs are already accessible...

I think there's another reason people do this and I think it was an unfortunate 1.x workaround turned 2.x anti-pattern. I think some people use job logs as a form of _output_. So something the job wrote to its logs might be retrieved and used as input (or as a conditional) as the worker continues doing whatever it does.

That feels dirty to me, but I know a lot of people did it because they _couldn't_ do something better-- they couldn't have a job write to a file that was accessible to the worker because in v1.x, shared storage was only mounted to the jobs and _not_ mounted to the worker itself. But that's fixed in 2.0, so I'm questioning whether there's any reason at all to allow workers to access job logs.

That said... I am not necessarily intent on removing `job.logs()` right now, but I'm holding back on documenting / demonstrating it for now.